lib/rouge/lexers/perl6.rb

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8 -*-
+
+module Rouge
+  module Lexers
+    class Perl6 < Perl
+      title "Perl 6"
+      desc "The Perl 6 programming language (perl6.org)"
+
+      tag "perl6"
+      aliases 'pl6'
+
+      filenames '*.pl6', '*.pm6'
+      mimetypes 'text/x-perl6', 'application/x-perl6'
+
+      def self.analyze_text(text)
+        return 1 if text.shebang? 'perl6'
+        return 0.4 if text.include? 'use v6'
+      end
+
+      hyper_operators = %w(« » << >>)
+
+      operator_words = %w(
+        div xx x mod also leg cmp before after eq ne le lt not gt eqv
+        ff fff min max not so and andthen or xor orelse extra lcm gcd o
+      )
+
+      pre_declares = %w(multi proto only)
+
+      declares = %w(
+        macro sub submethod method category module class role package
+        enum grammar slang subset
+      )
+
+      rules = %w(regex rule token)
+
+      includes = %w(use require\ unit)
```

Comment:
Typo in `require/`?

Reply:
I copied this from existing [Emacs perl6-mode rules](https://github.com/perl6/perl6-mode/blob/master/perl6-font-lock.el#L159), but checking others like [pygments' Perl6Lexer](https://bitbucket.org/birkenfeld/pygments-main/src/7941677dc77d4f2bf0bbd6140ade85a9454b8b80/pygments/lexers/perl.py?at=default#perl.py-243) seems to be more correct.  I'll likely change it as I further flesh out the RegexLexer-based update 👍